packages/react/README.md

```diff
@@ -54,14 +58,71 @@ createRoot(document.getElementById("root")!).render(
 );
 ```
 
-### 3. Importing and rendering components
+### 3. Using native Synergy components in react (only for react >= 19.0.0) in Typescript projects
+
+React@19 finally shipped with official support for web components.
+With this version of react, you are free to **use our native web components** directly in your application.
+
+However, you will likely receive errors because our elements are not known to React as available (in react speech `intrinsic`) elements. This will also occur when using typescript. For this reason, we provide **type only wrappers** for all versions of react from version 19.0.0 onward.
+
+Using synergy in a typescript project with React@19 can be easily achieved via one line of code. There is no need to import `@synergy-design-system/react` in your code directly anymore!
+
+Just add the following definition to your projects typescript configuration file (e.g. `tsconfig.json`):
+
+```json
+{
+  "compilerOptions": {
+    "types": ["@synergy-design-system/react/types/latest"]
+  }
+}
+```
+
+This makes sure your project knows about our list of intrinsic elements. This will also enable **automatic type checks and auto completion for properties** for all synergy elements.
+
+You may now use the components by importing them from the `@synergy-design-system/component` package and rendering them in a React component.
+
+```tsx
+// You may also load the complete bundle somewhere in your application,
+// but directly including only needed components leads to smaller bundles.
+import "@synergy-design-system/components/components/button/button.js";
+import "@synergy-design-system/components/components/input/input.js";
+
+export const MyButton = () => <syn-button type="submit">Submit me</syn-button>;
+export const MyInput = () => (
+  <syn-input name="my-input" onsyn-change={e => console.log(e)} required />
+);
+```
+
+#### 3.1. Migrating from synergies react wrappers to native components
+
+> Notice! Due to a [currently open bug in lits react wrapper in version 1.0.6](https://github.com/lit/lit/issues/4857), you will have to provide an override for the react types temporarily. This can be archived by adding the following override in your `package.json` file:
```

Comment:
archived --> achieved ;) 
```suggestion
> Notice! Due to a [currently open bug in lits react wrapper in version 1.0.6](https://github.com/lit/lit/issues/4857), you will have to provide an override for the react types temporarily. This can be achieved by adding the following override in your `package.json` file:
```